### PR TITLE
CORDA-2654: Repair the DJVM CLI tool.

### PR DESCRIPTION
**Pull request by chrisr3**
Realign the DJVM CLI with the more recent DJVM work so that it can build and run classes based on `java.util.function.Function` rather than its own `SandboxedRunnable`.

### CODE REVIEW

File: docs/source/key-concepts.rst [hunk 1]
```diff
@@ -24,7 +24,6 @@ This section should be read in order:
    key-concepts-node
    key-concepts-tearoffs
    key-concepts-tradeoffs
-   key-concepts-djvm
```
Comment (tlil) [inline docs/source/key-concepts.rst]: Why is this pulled out?

Comment (chrisr3) [reply docs/source/key-concepts.rst]: Because I thought we were pulling the DJVM from the docs?

Comment (tlil) [reply docs/source/key-concepts.rst]: That was discussing the _release notes_, not this section. And I believe we've gone back on that now, with this PR.

**Comment by chrisr3** [comment]
Actually, I think the line about "logging an informational message" no longer applies. This was previously implemented via a `Rule` but I don't seem able to trigger it any more.

**Pull request merged**